packages/stark-ui/src/modules/svg-view-box/directives/svg-view-box.directive.spec.ts

```diff
@@ -106,14 +104,15 @@ describe("SvgViewBoxDirective", () => {
 			initializeComponentFixture();
 		});
 
-		it("should keep the viewBox attribute as is", () => {
+		// tslint:disable-next-line:no-identical-functions
+		it("should overwrite the viewBox attribute", () => {
 			expect(fixture).toBeDefined();
 			const parentElement: DebugElement = fixture.debugElement.query(By.directive(StarkSvgViewBoxDirective));
 			expect(parentElement).toBeDefined();
 			const svgElement: SVGElement = parentElement.nativeElement.querySelector("svg");
 			expect(svgElement).toBeDefined();
 			expect(svgElement.hasAttribute("viewBox")).toBe(true);
-			expect(svgElement.getAttribute("viewBox")).toBe(viewBoxAttribute);
+			expect(svgElement.getAttribute("viewBox")).toBe("0 0 24 24");
```

Comment:
Can you re-use the `DEFAULT_VIEW_BOX_SIZE` constant instead of putting the hard-coded values in here?